functions/generate-pdf/2.0/function.json

```diff
@@ -0,0 +1,89 @@
+{
+  "description": "Generate a PDF based on HTML and CSS",
```

Comment:
@Reinier1982 maybe we can mention in this description that its based on LiquidJS. Something like:

`Generate a PDF based on HTML, CSS and LiquidJS`